ws-nextjs-app/pages/[service]/send/[id]/send.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PageLayoutWrapper from '#app/components/PageLayoutWrapper';
+import postFixture from './fixture';
+import UGCPage from './UGCPageLayout';
+
+const Component = () => (
+  // @ts-expect-error partial data required for storybook
+  <PageLayoutWrapper pageData={postFixture} status={200}>
+    <UGCPage pageData={postFixture} />
```

Comment:
Rename postFixture = perhaps `formFixture` or `ugcPageData`?

```suggestion
import formFixture from './fixture';
import UGCPage from './UGCPageLayout';

const Component = () => (
  // @ts-expect-error partial data required for storybook
  <PageLayoutWrapper pageData={formFixture} status={200}>
    <UGCPage pageData={formFixture} />
```

OR 

```suggestion
import ugcPageData from './fixture';
import UGCPage from './UGCPageLayout';

const Component = () => (
  // @ts-expect-error partial data required for storybook
  <PageLayoutWrapper pageData={ugcPageData} status={200}>
    <UGCPage pageData={ugcPageData} />
```